src/js/view/controls/controls.js

```diff
@@ -126,9 +126,10 @@ export default class Controls {
 
         // Settings Menu
         let lastState = null;
-        const visibilityChangeHandler = (visible, isKeyEvent) => {
+        const visibilityChangeHandler = (visible, evt = {}) => {
```

Comment:
Not a required change, but I still prefer doing an `undefined` check manually because of the output when a default argument is transpiled.